task4.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+using namespace std;
+struct Students {
+    char name[30];
```

Comment:
30 should be const, also it should be 31 because of the \0.
Create const MAX_NAME_LENGTH = 30 and use it here as +1